### ISSUE TITLE
"Copy As"/"Copy Remote File Url" copies %23 instead of # in case of Gitea

### ISSUE THREAD

**Issue by Eugeny48** [gitkraken/vscode-gitlens#2252]
### Description

my.code-workspace
```JSON
{
	"folders": [
	],
	"settings": {
		"gitlens.remotes": [{
			"domain": "my.com",
			"type": "Gitea"
		}]
	}
}
```

For github repos "Copy Remote File Url" works fine, but for Gitea repos the command replaces "#" character with "%23":
```
https://my.com/project/src/commit/00000000000000000/repo/folder/file.txt%23L20
```
But it should be
```
https://my.com/project/src/commit/00000000000000000/repo/folder/file.txt#L20
```

### GitLens Version

v12.2.2

### VS Code Version

Version: 1.72.0 (user setup)
Commit: 64bbfbf67ada9953918d72e1df2f4d8e537d340e
Date: 2022-10-04T23:20:39.912Z (2 days ago)
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.19044
Sandboxed: No

### Git Version

git version 2.29.2.windows.2

**Comment by eamodio** [comment]
Can you please verify this fix in tomorrow's pre-release edition (2023.3.30) of GitLens? Thank you so much!

You can switch to the pre-release edition of GitLens, by clicking on the "Switch to Pre-Release version of this extension" from the Extensions view.

![image](https://user-images.githubusercontent.com/641685/199397587-6948b1bc-3019-4d85-a6d5-332fea5d8f78.png)

**Comment by Eugeny48** [comment]
> Can you please verify this fix

It works for me, thank you.
